File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/PlusApiResultShareRecordVO.kt
package com.sdkwork.app

data class PlusApiResultShareRecordVO(
    val data: ShareRecordVO? = null,
    val code: String? = null,
    val msg: String? = null,
    val requestId: String? = null,
    val ip: String? = null,
    val hostname: String? = null,
    val errorName: String? = null
)

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/plus_api_result_share_record_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusApiResultShareRecordVO:
    """API调用结果"""
    data: ShareRecordVO
    code: str
    msg: str
    request_id: str
    ip: str = None
    hostname: str = None
    error_name: str
